test/verify/check-storage-mounting

```diff
@@ -69,9 +69,27 @@ class TestStorageMounting(StorageCase):
 
         self.content_tab_wait_in_info(1, 1, "Mount point", mount_point_foo)
 
+        # Keep the mount point busy
+        sleep_pid = m.spawn(f"cd {mount_point_foo}; sleep infinity", "sleep")
+        self.write_file("/etc/systemd/system/keep-mnt-busy.service",
+                        f"""
+[Unit]
+Description=Test Service
+
+[Service]
+WorkingDirectory={mount_point_foo}
+ExecStart=/usr/bin/sleep infinity
+""")
+        m.execute("systemctl start keep-mnt-busy")
+
         self.content_dropdown_action(1, "Unmount")
+        b.wait_in_text("#dialog", str(sleep_pid))
```

Comment:
Please also assert that keep-mnt-busy service is mentioned in the dialog

Reply:
Done.